crates/viewer/re_viewer_context/src/gpu_bridge/outline.rs

```diff
@@ -0,0 +1,24 @@
+//! Utilities related to outlines.
+
+use egui::NumExt as _;
+
+use re_renderer::OutlineConfig;
+use re_ui::ContextExt;
+
+/// Produce an [`OutlineConfig`] based on the [`egui::Style`] of the provided [`egui::Context`].
+pub fn outline_config(gui_ctx: &egui::Context) -> OutlineConfig {
```

Comment:
gpu bridge kinda makes sense, but technically we only need this on re_space_view
Unless we think that non-view crates need this as well 🤔